_notifications-plugin/index.md

```diff
@@ -0,0 +1,135 @@
+
+The notifications plugin currently supports two ways of user authentication:
+
+1. Provide the user with full access to Amazon SNS.
+2. Let the user assume an IAM role that has permissions to access Amazon SNS. Once you configure the notification channel to use the right Amazon SNS permissions, select the OpenSearch plugins that can trigger notifications.
+
+### Provide full Amazon SNS access permissions
+
+If you want to provide full Amazon SNS access to the IAM user, ensure that the user has the following permissions.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "sns:*"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+### Assuming an IAM role with Amazon SNS permissions
+
+If you want to let the user send notifications without directly having full permissions to Amazon SNS, let the user assume a role that does have the necessary permissions.
+
+The IAM user must have the following permissions to assume a role.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:Describe*",
+        "iam:ListRoles",
+        "sts:AssumeRole"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+```
+
+Then add this policy into the IAM user’s trust relationship to actually assume the role.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+  {
+    "Effect": "Allow",
+    "Principal": {
+    "AWS": "arn:aws:iam::<arn_number>:user/<iam_username>",
+    },
+    "Action": "sts:AssumeRole"
+  }
+  ]
+}
+```
+
+
+## Email as a channel type
+
+To send or receive notifications with emails, choose **Email** as the channel type. Next, select at least one sender and default recipient. To send notifications to more than a few people at a time, select a recipient group. If the Notifications plugin doesn’t currently have the necessary senders or groups, you can add them by first selecting **SMTP sender**, then choose **Create SMTP sender** or **Create recipient group**. Choose **SES sender** to use Amazon Simple Email Service (SES).
+
+### Create email sender
+
+1. Specify a unique name to associate with the sender.
+2. Enter an email address, and, if applicable, its host (for example, smtp.gmail.com), and the port. If you're using SES, enter the IAM role ARN of the AWS account to send notifications from, along with the region.
+3. Choose an encryption method. Most email providers require SSL or TLS, which requires a username and password in the OpenSearch keystore. See [Authenticate sender account](#authenticate-sender-account) to learn more. Selecting an encryption method is only applicable if you're creating an SMTP sender.
```

Comment:
Please define Secure Sockets Layer (SSL) and Transport Layer Security (TLS). Change "requires" to "require". Change "username" to "user name".